apps/design/frontend/src/app.tsx

```diff
@@ -69,6 +76,23 @@ function ElectionScreens(): JSX.Element {
   );
 }
 
+function WaitForUserInfo(props: { children: React.ReactNode }) {
+  const { children } = props;
+
+  const userLoaded = getUser.useQuery().isSuccess;
+  if (!userLoaded) {
+    return (
+      <Screen>
+        <Main centerChild>
+          <LoadingAnimation />
+        </Main>
+      </Screen>
+    );
+  }
+
+  return children;
+}
```

Comment:
Is it generally slow enough loading the user that we see the loading animation?

Reply:
Yeah, good question - it was, occasionally, but might just be a dev thing. Might end up being better to leave it out in prod to avoid flicker; wasn't sure what latency spikes/user connection quirks to expect in practice, but those should be out of the ordinary anyway - will make a note to revisit